task/upload.go

```diff
@@ -467,6 +457,7 @@ func outputLocations(outURL string, relativePath string) ([]OutputName, []client
 				Outputs: &clients.OutputsRequest{
 					SourceSegments:     sourceSegments,
 					TranscodedSegments: true,
+					AutoMP4:            true,
```

Comment:
This will set `AutoMP4` not only for `upload` but also for `transcode-file` tasks. I'd suggest receiving the `autoMP4` value as a boolean argument in this function, then setting it to true/false depending on where you call it from.

While at it, we could rename `assetOutputLocations` to `uploadTaskOutputLocations` to be clearer, wdyt?

Reply:
Oh right, makes sense.
And sure for the renaming!